lib/rubyvm/debug_inspector.rb

```diff
@@ -0,0 +1,6 @@
+require 'rbconfig'
+dlext = RbConfig::CONFIG['DLEXT']
+begin
+  require "rubyvm/debug_inspector.#{dlext}"
+rescue LoadError
```

Comment:
The reason I ask is because if this fails, `RubyVM::DebugInspector.open` resolves to a private method from elsewhere. But the cause isn't obvious.